modules/amf-registration/amf-registration-service/service.xml

```diff
@@ -29,11 +29,11 @@
 		<column name="emailAddress" type="String" />
 		<column name="userName_" type="String" />
 		<column name="gender" type="String" />
-		<column name="birthday" type="Date" />
+		<column name="birthday" type="String" />
```

Comment:
you could leave as a `Date` here and `String` in the rest api, and use a parse function to convert from `String` to `Date`.